hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -1359,6 +1359,10 @@ public HoodieTableVersion getWriteVersion() {
     return HoodieTableVersion.fromVersionCode(getIntOrDefault(WRITE_TABLE_VERSION));
   }
 
+  public void setWriteVersion(HoodieTableVersion version) {
+    setValue(WRITE_TABLE_VERSION, String.valueOf(version.versionCode()));
+  }
```

Comment:
Avoid adding setters here and only add the setter to the `HoodieWriteConfig.Builder`.

Reply:
I can make this change but i see this pattern in other areas of `HoodieWriteConfig`.  https://github.com/apache/hudi/blob/master/hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java#L1333